### ISSUE TITLE
Gradle plugin portal submission rejected multiple times

### ISSUE THREAD

**Issue by donat** [Yeregorix/DependencyExporter#1]
Hi! We have received multiple submissions to the Gradle plugin portal for the `net.smoofyuniverse.dependency-exporter` plugin. Unfortunately, we could not accept them due to some validation errors. However, we've sent rejection emails with the errors and the steps to correct them. I'm opening this issue to verify whether you've received the emails.

**Comment by Yeregorix** [comment]
Hi, I receive your emails, except one time when I got rejected without email but every other times I got them. I wasn't sure if I could reply to your emails as I thought it was a semi-auto process.

I tried to prove ownership one time with TXT record:
name : `gradle` and value : `gradle-verification=FAF11932481C7948C769D4818E6F2`
And a second time with:
name : `gradle-verification` and value : `FAF11932481C7948C769D4818E6F2`

Each time I've waited more than 1 hour before submitting (the TTL of my records) but none worked.
Maybe I've done something wrong. If you accept, I can also send you a mail from `admin` `@` `smoofyuniverse` `.` `net` to prove ownership.
